keyboards/arabica37/keymaps/default/glcdfont.c

```diff
@@ -0,0 +1,232 @@
+// This is the 'classic' fixed-space bitmap font for Adafruit_GFX since 1.0.
```

Comment:
Is there a reason for having this in the default keymap folder, as opposed to the keyboard's folder? 

Mostly, because in it's current location, it's likely to get duplicated repeatedly. 

If you could, please move it to keyboard folder.

Reply:
I moved it. Is this ok?